tests/integration_tests/api/system_backend/test_tools.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from unittest import TestCase
+
+from parameterized import parameterized, param
+
+from hvac import exceptions
+from tests.utils.hvac_integration_test_case import HvacIntegrationTestCase
+
+
+class TestTools(HvacIntegrationTestCase, TestCase):
+    @parameterized.expand(
+        [
+            param(
+                "success",
+            ),
+        ]
+    )
+    def test_generate_random_bytes(
```

Comment:
looks like we're not testing the `source` parameter, let's add some test cases and/or a new test method for testing the various `source` options